src/Components/SmartComponents/Modals/ReportConfigModal.js

```diff
@@ -137,7 +137,7 @@ const ReportConfigModal = ({ isOpen, onClose }) => {
                         label={intl.formatMessage(messages.customOnlyCvesWithRulesLabel)}
                         isChecked={filterData.security_rule === 'true'}
                         onChange={(newValue) =>
-                            setFilterData({ ...filterData, security_rule: newValue ? 'true' : 'all' })
+                            setFilterData({ ...filterData, security_rule: newValue ? 'true' : 'true,false' })
```

Comment:
ooops, `security_rule` should be `rule_presence` ? 
Also, I think instead of passing `true,false` to get all the data you pass 'undefined' and the APIHelper should omit that param filter so in the end you should expect all the data

Reply:
Well the security_rule was passed into rule_presence inside ReportHelper, so it worked, but here I see that's confusing so I'm refactoring to rule_presence.

Reply:
I intentionally left security_rule because it references FILTERS object to get filter name, which uses security_rule not rule_presence.